### ISSUE TITLE
Split tunneling import crashes the app (Windows client)

### ISSUE THREAD

**Issue by nlobanov** [amnezia-vpn/amnezia-client#417]
**Describe the bug**
I imported a huge json of 25 megabytes. After importing, amnezia vpn froze. After killing it and restarting, I can no longer open the split tunneling page, the client just freezes when opening it. RAM consumption is rapidly increasing, starting at 1-2 gigabytes and filling 100% of my RAM (16gb) within 10-20 seconds.

**To Reproduce**
1. Go to Split tunneling options
2. Import large json with urls
4. Amnezia client is not responding

**Log files**
In the archive there is a json file containing all url's blocked by roskomnadzor. Json in the format that amnezia vpn generates when saving the list. 
P.S. yes, I know it's probably not the smartest idea to import such a large json.
[transformed_websites.zip](https://github.com/amnezia-vpn/amnezia-client/files/13256614/transformed_websites.zip)

**Screenshots**
![image](https://github.com/amnezia-vpn/amnezia-client/assets/26668110/80278428-3e76-40c7-a121-a19da6f8f955)
![image](https://github.com/amnezia-vpn/amnezia-client/assets/26668110/945db6f4-6de9-4b98-a1e0-1685032a3c52)

**Expected behavior**
I expect the json file to either be processed or I get an error.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 4 0 8 6

**Comment by outspace** [comment]
1) We made a rework of Split tunnel. In version 4.1.0.0 we added WG/AWG support for the desktop and some other fixes.
It can be downloaded from [https://github.com/amnezia-vpn/amnezia-client/actions/runs/6750374498](https://github.com/amnezia-vpn/amnezia-client/actions/runs/6750374498)

2) Your JSON list is wrong. It operates with IP, and you must set up "ip" filed. 

3) List can be optimized by subnets. It supports values like 192.168.31.0/24. Proper way to make this list is to find all blocked ip ranges, than build json list of that ranges.
